src/components/input/TypeHandler/Base.ts

```diff
@@ -124,6 +121,7 @@ export class Base extends TypeHandler {
 					}
 			}
 		}
-		return this.formatState(result)
+		const r = this.formatState(StateEditor.copy(result))
```

Comment:
Should use the renamed version of:
```
return this.formatState(StateEditor.copy(result)).stateCopy
```

Reply:
.formatState already returns a state obtained with .stateCopy though. But I'll get rid of the unnecessary line.

Reply:
Yes, your're right.